packages/core/src/basebackend.ts

```diff
@@ -91,9 +97,23 @@ export abstract class BaseBackend<O extends Options> implements Backend {
    * @inheritDoc
    */
   public sendEvent(event: Event): void {
-    void this._transport.sendEvent(event).then(null, reason => {
-      isDebugBuild() && logger.error('Error while sending event:', reason);
-    });
+    // TODO(v7): Remove the if-else
+    if (
+      this._newTransport &&
+      this._options.dsn &&
+      this._options._experiments &&
+      this._options._experiments.newTransport
+    ) {
+      const api = initAPIDetails(this._options.dsn, this._options._metadata, this._options.tunnel);
```

Comment:
it's annoying to do this, but considering that the client has access to API, won't be needed in the future!